### ISSUE TITLE
Adding TRPC

### ISSUE THREAD

**Issue by OliverGeneser** [RobSchilderr/capacitor-supertokens-nextjs-turborepo#11]
I'm working on implementing TRPC into the project, is it in the scope of the project? If so I will create a PR when I'm done :slightly_smiling_face:

**Comment by RobSchilderr** [comment]
Hey Oliver! Yes it is, but I think it should be another repo because this one should be less opinionated and really serve as an example. We can make Next Native TRPC together, I can also work on it if necessary.

**Comment by OliverGeneser** [comment]
I agree lets keep this repo clean and create a new one. Lets work together 👍

**Comment by RobSchilderr** [comment]
> I agree lets keep this repo clean and create a new one. Lets work together 👍 

I'll aim for making the repo tomorrow

**Comment by RobSchilderr** [comment]
@OliverGeneser the [Next.tjs Native TRPC repository](https://github.com/RobSchilderr/nextjs-native-trpc) has been made! Looking forward to your contributions. You can check [this PR](https://github.com/timothymiller/t4-app/pull/106) as an example on how to work with Supertokens and TRPC.

**Comment by OliverGeneser** [comment]
@RobSchilderr I have totally missed the notification... I'm a little busy this weekend but I will try to move my changes into the new repo in the next week or so.
If you want you can look at my changes in nextjs-native-starter-geneser/tree/trpc, the only thing I'm missing, is getting the Supertokens session in the TRPC API
